### ISSUE TITLE
[Priest] Learning Shadowguard Rank 4 not possible

### ISSUE THREAD

**Issue by uelf1** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3851]
Report ID: N/A

Reporter Name: Kochbuch (on Archangel)

Date/Time: 29.08.2016 10:40

Error Severity: N/A

Zone/Instance: N/A

What type of error occurred?: I can not learn **Shadowguard Rank4** at any Priest teacher.

What was the actual result vs. the expected result?: I'm Stuck at **Rank 3** instead of **Rank 5** or above.

Can the error be repeated? If so, what are the steps?: unknown

Link to Video Evidence: N/A

External/Related Links: http://wowwiki.wikia.com/wiki/Shadowguard

Please detail your character’s race, class, and items (optional): Troll Priest level 61 at the moment. Character name: Kochbuch

Details: I went to every priest trainer and no one could teach me **Shadowguard Rank 4** because it doesn't exist. **Rank 5** and above are shown to learn, but I need to learn **Rank 4** before learning **Rank 5**, but that is not possible. I asked other troll priests and they told me that they don't have this issue. I already cleared my Cache and WTF folder. Any suggestions on what I should do or can you fix this? Thanks!

**Comment by lajhar** [comment]
Cannot reproduce it. They teach every rank (1-7) for me as intended

**Comment by uelf1** [comment]
now i'm lvl 70 and i could suddenly learn shadowguard rank4 and above, but before lvl 70 it was not possible.

**Comment by lajhar** [comment]
In this case, issue confirmed, thank you for your report, I wasn't thorough enough.

[Shadowguard](http://wowwiki.wikia.com/wiki/Shadowguard) (Rank 4) wrongly **requies level 68** to be learnt.
![image](https://cloud.githubusercontent.com/assets/20255946/18089373/f0965008-6ebf-11e6-945a-d659b78f2e6f.png)

Expected rank table:

- 1: Level 20
- 2: Level 28
- 3: Level 36
- **4: Level 44**
- 5: Level 52
- 6: Level 60
- 7: Level 68

**Comment by uelf1** [comment]
yeah that is strange. I hope it can be fixed. Because Rank 5 requires level 52 or something like that.
I hope it is an easy bug to fix.  Thanks for admitting :)

**Comment by devastator** [comment]
Fixed. Soon on PTR

**Comment by lajhar** [comment]
Fix confirmed on PTR.
